### DISCUSSION TITLE
Accessing Peers From Outside the wg-easy Container

### DISCUSSION THREAD

**Discussion by brandonegg** [wg-easy/wg-easy#807] [Q&A] [answered]
I'm curious if there's a step I have forgotten or if this is expected. Currently the only way I can access peers on my wireguard network from the "host" machine (in this context host refers to the machine running the wg-easy docker container) from outside the container itself.

For example: I have a peer on my wireguard network with ip 10.8.0.4, if I run `docker exec -it {wireguard-container-name} /bin/sh` and within the container run `ping 10.8.0.4` the packets are sent successfully. However, if I now exit the container and try running `ping 10.8.0.4` the packets fail to send.

Is there a way that wg-easy can be run as a UI on top of the system wireguard? I would think that setting the docker container network to host would resolve this issue maybe? I'm not quite sure and was wondering if anyone else had this issue and if/how they solved it? Thanks!

**Comment by eandersons** [comment, marked as answer]
After a little search I found that the discussion #508 and the issue #729 contains some information and possible solutions regarding this.

**Reply by brandonegg** [reply]
@eandersons Thank you so much! I had been searching quite some time for something like this but I guess I'm a bad googler.
